homeassistant/components/slide_local/config_flow.py

```diff
@@ -136,6 +136,46 @@ async def async_step_user(
             errors=errors,
         )
 
+    async def async_step_reconfigure(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Handle reconfiguration of the integration."""
+        errors: dict[str, str] = {}
+
+        if user_input is not None:
+            if not (errors := await self.async_test_connection(user_input)):
+                await self.async_set_unique_id(self._mac)
+                self._abort_if_unique_id_mismatch(
+                    description_placeholders={CONF_MAC: self._mac}
+                )
+                user_input |= {
+                    CONF_API_VERSION: self._api_version,
+                }
+
+                return self.async_update_reload_and_abort(
+                    self._get_reconfigure_entry(),
+                    data_updates=user_input,
+                )
+
+        entry: SlideConfigEntry = self._get_reconfigure_entry()
+
+        return self.async_show_form(
+            step_id="reconfigure",
+            data_schema=self.add_suggested_values_to_schema(
+                vol.Schema(
+                    {
+                        vol.Required(CONF_HOST): str,
+                        vol.Optional(CONF_PASSWORD): str,
+                    }
+                ),
+                {
+                    CONF_HOST: entry.data[CONF_HOST],
+                    CONF_PASSWORD: entry.data.get(CONF_PASSWORD, ""),
+                },
+            ),
```

Comment:
So the device we connect with is always the same, so as far as we know (from the manufacturer), the password can't change, so in theory we could just only ask for the HOST here.

(If there will be reports of people being able to change passwords we can always update this, that's not a hard fix)